lib/model/rwfolder.go

```diff
@@ -847,18 +835,7 @@ func (f *sendReceiveFolder) deleteFile(file protocol.FileInfo) {
 		return
 	}
 
-	cur, ok := f.model.CurrentFolderFile(f.folderID, file.Name)
-	if ok && f.inConflict(cur.Version, file.Version) {
```

Comment:
Good catch. I must have screwed that up when I was copy'n'pasting things around. I added this to the deleteFile.